.circleci/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM    circleci/openjdk:latest
+
+# Based on https://github.com/GoogleCloudPlatform/cloud-builders/blob/master/bazel/Dockerfile
+RUN \
+    sudo sh -c 'echo "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8" > /etc/apt/sources.list.d/bazel.list '
+
+RUN \
+    curl https://storage.googleapis.com/bazel-apt/doc/apt-key.pub.gpg | sudo apt-key add -
+
+RUN sudo apt-get update && \
```

Comment:
I think you want to stick to a version as we don't always want to use the latest.

Reply:
I'm not sure - in general the latest is supposed to be better :-) I would rather find problems with new bazel earlier. We do need to tag the docker images used for releases, so we can repro, and of course bazel must take backward compatibility seriously.